tsconfig.eslint.json

```diff
@@ -1,4 +1,10 @@
 {
   "extends": "./tsconfig.base.json",
-  "include": ["**/*.js", "**/*.ts", "**/*.tsx", ".eslintrc.js"]
-}
+  "include": [
+    "**/*.js",
+    "**/*.ts",
+    "**/*.tsx",
+    ".eslintrc.js",
+    "*.svg"
```

Comment:
Is this line needed for code to work properly?

Reply:
removed